core/templates/dev/head/components/state-editor/state-editor-properties-services/state-content.service.ts

```diff
@@ -15,12 +15,25 @@
 /**
  * @fileoverview A data service that stores the current state content.
  */
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
 
+import { AlertsService } from 'services/alerts.service';
+import { StatePropertyService } from
+  // eslint-disable-next-line max-len
+  'components/state-editor/state-editor-properties-services/state-property.service';
```

Comment:
Can't we just split this using a `+`, or is that not possible here? Ditto elsewhere.

Reply:
No, this isn't possible here.

Reply:
Ok, thanks for the clarification!